interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -641,92 +633,56 @@ int acados_create() {
         forw_vde_casadi[i].casadi_sparsity_in = &{{ model.name }}_expl_vde_forw_sparsity_in;
         forw_vde_casadi[i].casadi_sparsity_out = &{{ model.name }}_expl_vde_forw_sparsity_out;
         forw_vde_casadi[i].casadi_work = &{{ model.name }}_expl_vde_forw_work;
-        {% if ocp.dims.np < 1 %}
-        external_function_casadi_create(&forw_vde_casadi[i]);
-		{% else %}
-        external_function_param_casadi_create(&forw_vde_casadi[i], {{ocp.dims.np}});
-		{% endif %}
+        external_function_param_casadi_create(&forw_vde_casadi[i], {{dims.np}});
```

Comment:
as a general rule, we could stick to {{ <something> }}, with spaces after {{ and before  }}, for better readability.

Reply:
alright.